package.json

```diff
@@ -29,5 +29,9 @@
     "url": "http://www.bentley.com"
   },
   "//dependencies": "NOTE: dependencies will be empty here and instead specified in each package.",
-  "//devDependencies": "NOTE: devDependencies will be empty here and instead specified in each package."
-}
\ No newline at end of file
+  "//devDependencies": "NOTE: devDependencies will be empty here and instead specified in each package.",
+  "packageManager": "pnpm@10.8.0+sha512.0e82714d1b5b43c74610193cb20734897c1d00de89d0e18420aebc5977fa13d780a9cb05734624e81ebd81cc876cd464794850641c48b9544326b5622ca29971",
+  "dependencies": {
```

Comment:
you can remove these changes, and then run rush install so the pnpm-lock.yaml file can go back to the original